services/argus/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v0.9.7 (2024-04-11)
+
+- Add Middleware logic to the client.
```

Comment:
Make it more explicit:
"Support `WithMiddleware` configuration option in the client"